rpc/glfs-operations.c

```diff
@@ -212,7 +213,11 @@ glusterBlockCreateEntry(struct glfs *glfs, blockCreateCli *blk, char *gbid,
     ret = -1;
     goto out;
   } else {
+#if GFAPI_VERSION760
```

Comment:
@lxbsz 
Just a cosmic suggestion to make it more opt for reader, while the code looks good to me.

```
#ifdef HAVE_LEGACY_FTRUNCATE_GFAPI
    ret = glfs_ftruncate(tgfd, blk->size);
#else
     ret = glfs_ftruncate(tgfd, blk->size, NULL, NULL);
#end
```
Thanks!

Reply:
Prasanna, while for this one call it looks good, what if we have to add more `glfs_*` calls later?

Reply:
@amarts AFAIK in 760 ftruncate (and preadv/pwritev which we don't use ATM) signatures got changed ? so if other glfs_* calls change in future we need to redefine GFAPI_VERSION_* again anyway. 

But it was just a cosmic suggestion. I'm okay both way. Feel free to say "lets live with it"

Reply:
Yeah, this is why I use the 760 instead, if we have new changes again for other APIs.

Reply:
Alright +1 from me too :-)